apps/server/src/apps/board-collaboration.app.ts

```diff
@@ -18,19 +18,29 @@ import {
 	createAndStartPrometheusMetricsAppIfEnabled,
 } from '@src/apps/helpers/prometheus-metrics';
 import { ExpressAdapter } from '@nestjs/platform-express';
+import { DB_URL } from '@src/config';
+import { IoAdapter } from '@nestjs/platform-socket.io';
 
 async function bootstrap() {
 	sourceMapInstall();
 
 	const nestExpress = express();
 	const nestExpressAdapter = new ExpressAdapter(nestExpress);
 	const nestApp = await NestFactory.create(BoardCollaborationModule, nestExpressAdapter);
-	nestApp.useLogger(await nestApp.resolve(LegacyLogger));
+	const legacyLogger = await nestApp.resolve(LegacyLogger);
+	nestApp.useLogger(legacyLogger);
 	nestApp.enableCors({ exposedHeaders: ['Content-Disposition'] });
 
-	const mongoIoAdapter = new MongoIoAdapter(nestApp);
-	await mongoIoAdapter.connectToMongoDb(DB_URL);
-	nestApp.useWebSocketAdapter(mongoIoAdapter);
+	let ioAdapter: IoAdapter;
+	// eslint-disable-next-line no-process-env
+	if (process.env.COLLABORATION_WEBSOCKET_ADAPTER === 'redis') {
```

Comment:
Any reason not to use the configuration module here?

Reply:
fixed. removed configurability.